README.md

```diff
@@ -1,2 +1,218 @@
+
+## 🛠 Built With  <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Medium-fidelity Wireframes]**
+- **[Desktop Version]**
+- **[Dynamic Design]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎥 Video Explanation <a name="video-explanation"></a>
+Coming soon!
+
+## 🚀 Live Demo <a name="live-demo"></a>
+Coming Soon!
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your first "Awesome Books" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+-Git bash
+-Node JS
+-Visual Studio Code as your code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone[(https://github.com/AtifZada/Awesome-Books.git)]
+```
+### Set Up Linters
+## Define Linters
+A linter is a tool to help you improve your code. You can learn more about Linters here: (source: (<https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/>)).
+
+Advantages of Linting:
+
+1. Fewer errors in production- The use of linters helps to diagnose and fix technical issues such as code smells. As a result, fewer defects make their way to production.
+2. Achieving a more readable and consistent style, through the enforcement of its rules.
+3. Having more secure and performant code.
+4. Having an objective and measurable assessment of code quality.
+5. Having fewer discussions about code style and aesthetic choices during code reviews.
+
+## Install Linters 
+You can find linters for most of the programming languages, e.g. Rubocop for Ruby or ESLint for JavaScript.
+
+Also, there are many ways you can integrate a linter in your workflow:
+
+-text editor plugin
+-GitHub Actions
+-GitHub apps
+
+## Set up Linters
+**Note:** The npm package manager is going to create a node_modules directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a .gitignore file and add node_modules to it:
+
+# .gitignore
+node_modules/
+
+## ESLint 
+Run 
+```
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+```
+
+## Web Hint 
+This is a customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+
```

Comment:
- [ ] The Getting started section should describe step by step what a common user should do to use your app. Do they need to install linters? Please 🙏🏽 edit your getting started section giving the correct instructions to use your app.

Reply:
Thank your for the helpful review I will obviously change it and will come up with solution